multicluster/charts/linkerd-multicluster-link/values.yaml

```diff
@@ -11,14 +11,22 @@ gateway:
     port: 4191
 # -- Log level for the Multicluster components
 logLevel: info
+# -- Node selectors for the Service mirror pod
+nodeSelector: {}
+# -- Resources for the Service mirror container
+resources: {}
 # -- Number of times update from the remote cluster is allowed to be requeued
 # (retried)
 serviceMirrorRetryLimit: 3
 # -- User id under which the Service Mirror shall be ran
 serviceMirrorUID: 2103
+# -- Name of the target cluster that's going to be linked
+targetClusterName: ~
```

Comment:
I think an empty string is a better default/unset value here than "~".  The use of "~" suggests that this symbol has some special meaning or use.

Reply:
Makes sense, fixed!